external-crates/move/move-compiler/src/expansion/translate.rs

```diff
@@ -51,7 +51,7 @@ impl<'env, 'map> Context<'env, 'map> {
         compilation_env: &'env mut CompilationEnv,
         module_members: UniqueMap<ModuleIdent, ModuleMembers>,
     ) -> Self {
-        let mut all_filter_alls = WarningFilters::new();
+        let mut all_filter_alls = WarningFilters::new_for_lib_code();
```

Comment:
Given how this is used now, maybe we should rename this `dependency_filter` or something